modules/notify/main/index.coffee

```diff
@@ -42,8 +41,10 @@ redraw = (manager) ->
   view = container.view('.hx-notification')
   view.enter (d) ->
     selection = @append('div')
+    optionalClass = if d.options.cssclass then " #{d.options.cssclass}" else ''
+
     selection
-      .class('hx-notification ' + d.options.cssclass)
+      .class("hx-notification#{optionalClass}")
```

Comment:
Resolves an issue with trailing spaces as well as notifications being given the `undefined` class